src/Examine.Lucene/Search/LuceneSearchQuery.cs

```diff
@@ -309,5 +311,48 @@ public IBooleanOperation SelectAllFieldsInternal()
 
         protected override LuceneBooleanOperationBase CreateOp() => new LuceneBooleanOperation(this);
 
+        public override IFacetQueryField Facet(string field) => FacetInternal(field);
+
+        public override IFacetQueryField Facet(string field, string value) => FacetInternal(field, value);
+
+        public override IFacetQueryField Facet(string field, string[] values) => FacetInternal(field, values);
```

Comment:
Is it worth having this declared as `params string[] values` ? and same for the other array based parameter methods?

Reply:
I think this is a great idea. I've added it now with a null check where it's used.